tensorflow/tools/pip_package/BUILD

```diff
@@ -84,6 +84,7 @@ py_binary(
         "//tensorflow/python/saved_model",
         "//tensorflow/python:spectral_ops_test_util",
         "//tensorflow/python/tools:tools_pip",
+        "//tensorflow/python/eager:eager_pip",
```

Comment:
nit: @snnn We aim to keep this list alphabetically sorted. Can you fix the ordering?